cowprotocol/explorer
# [Batch Viewer] Show the receiver

# Summary

Closes #106 

Added receiver when it's different from the order's owner.

<img width="861" alt="Screen Shot 2022-06-14 at 10 02 20" src="https://user-images.githubusercontent.com/11525018/173583485-150f28fd-dacc-4ef5-a732-360714a76bfd.png">


# To Test

1. Open any tx with receivers and go to batch viewer tab. (i.e `tx/0x0d6f18a97690de9d70cdfe00231ee6baf722a12669f3260fb1faf3d2e25dca38`
* You will see the receiver grouped along with the owner of each order.

Example tx
Fix: https://pr120--explorer.review.gnosisdev.com/tx/0x0d6f18a97690de9d70cdfe00231ee6baf722a12669f3260fb1faf3d2e25dca38?tab=graph

Old: https://explorer.cow.fi/tx/0x0d6f18a97690de9d70cdfe00231ee6baf722a12669f3260fb1faf3d2e25dca38?tab=graph
